packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -35,14 +37,14 @@ typedef MessageFilter = String Function(String message);
 // DO NOT update without contacting kevmoo.
 // We have server-side tooling that assumes the values are consistent.
 class PubContext {
-  PubContext._(this._values) {
+  PubContext._(this._values) : assert(() {
```

Comment:
Why the move to an assert?

Reply:
I'd be concerned about an update or bug in this list breaking the tool if our test coverage isn't good. Moving to an assert lowers the risk.

On the other hand, its entirely unrelated to this PR, so I should remove it anyway